### PR TITLE
[FX-1828] Update typography and spacing in mobile menu

### PR DESCRIPTION
**Pull request by anandaroop**
https://artsyproduct.atlassian.net/browse/FX-1828

I picked off some of the purely visual work from the [parent ticket](https://artsyproduct.atlassian.net/browse/FX-1702).

The Figma designs include updates to typography (Unica) and spacing (tighter padding overall, and around the horizontal dividers.

Here's what that looks like:

## Before

before

## After

after

### CODE REVIEW

File: src/Components/NavBar/Menus/MobileNavMenu.tsx [hunk 1]
```diff
@@ -81,6 +81,8 @@ export const MobileNavMenu: React.FC<MobileNavMenuProps> = props => {
   )
 }
 
+export const MobileNavMenuSystem: React.FC = () => <h1>Hello, nav!</h1>
+
```
Comment (xtina-starr) [inline src/Components/NavBar/Menus/MobileNavMenu.tsx]: I think this was left in by mistake?

**Comment by xtina-starr** [comment]
I think `MobileNavMenuSystem` was left in by mistake. Outside of that, this looks good.

File: src/Components/NavBar/Menus/MobileNavMenu.tsx (hunk 1 above)
Comment (anandaroop) [reply src/Components/NavBar/Menus/MobileNavMenu.tsx]: Thank you, good catch! Will remove...

Comment (anandaroop) [reply src/Components/NavBar/Menus/MobileNavMenu.tsx]: Fixed! 🙈

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.21.4` :rocket:

**Pull request merged**